neqo-transport/src/recovery.rs

```diff
@@ -211,12 +228,102 @@ impl LossRecoverySpaces {
     }
 }
 
+#[derive(Debug)]
+struct CongestionControl {
```

Comment:
We're going to want to have different CC for each path.  But the loss recovery is going to report directly into CC for ACK/loss.  Maybe consider how that fits together.